tunder/lib/src/http/route_entry.dart

```diff
@@ -356,36 +333,30 @@ class RouteEntry implements RouteDefinitions {
 
     return method.parameters.map((param) {
       var type = param.type.reflectedType;
-      if (type == Request) return request;
-      if (_isPrimitive(type)) {
-        return paramValue(
-          param.simpleName,
-          request: request,
-          castTo: type,
-        );
-      }
 
-      return container.get(param.type.reflectedType);
+      if (type == Request) {
+        return request;
+      } else if (_isPrimitive(type)) {
+        return paramValue(param.simpleName, request: request, castTo: type);
+      } else {
+        return container.get(param.type.reflectedType);
+      }
     }).toList();
   }
 
-  bool _isPrimitive(type) {
-    return [int, double, String].contains(type);
-  }
+  bool _isPrimitive(type) => const [int, double, String].contains(type);
 
-  MethodMirror? _getMethodFrom(InstanceMirror instance, String methodName) {
-    return _getMethodsFrom(instance)
-        .where((element) => element.key == Symbol(methodName))
-        .toList()
-        .firstOrNull
-        ?.value as MethodMirror?;
-  }
+  MethodMirror? _getMethodFrom(InstanceMirror instance, String methodName) =>
+      _getMethodsFrom(instance)
+          .where((element) => element.key == Symbol(methodName))
+          .toList()
+          .firstOrNull
+          ?.value as MethodMirror?;
```

Comment:
Also, revert this one since it's a method with more than two lines of code. I think this is the pattern I want to follow to make the code style more readable a pleasant to read:

- Let's use arrow functions if the method has <= 2 lines of code. For any method with more than 2 lines of code we should use curly braces. Let me know if you have another opinion on this, I'm open to arguments.

Reply:
I feel like the final code isn't very different, it may be a matter of preference. The arrow notation removes some noise around the code and makes the code look more [Pythonic](https://imgur.com/a/rxQVNck). However, the usage of the arrow for complex functions is discouraged by the [Effective Dart guidelines](https://dart.dev/guides/language/effective-dart/usage#consider-using--for-simple-members). Overall, I believe the curly braces may look more natural to most people and the convention can be followed with ease by the contributors.

Reply:
I agree. I also don't like the Pythonic style. It's hard to read very long methods. 👍🏻